subxt/src/macros.rs

```diff
@@ -56,7 +56,7 @@ macro_rules! cfg_jsonrpsee_web {
 macro_rules! cfg_reconnecting_rpc_client {
 	($($item:item)*) => {
 		$(
-			#[cfg(all(feature = "unstable-reconnecting-rpc-client"))]
```

Comment:
nice catch

Reply:
I guess now we'll only use this if _some_ of the feature flag exists :'(